config/defaults/secure-open-banking/oauth2provider-update.json

```diff
@@ -187,17 +183,15 @@
   },
   "cibaConfig": {
     "supportedCibaSigningAlgorithms": [
-      "PS256",
-      "ES256"
+      "PS256"
     ],
     "cibaAuthReqIdLifetime": 600,
     "cibaMinimumPollingInterval": 2
   },
   "consent": {
     "clientsCanSkipConsent": false,
     "supportedRcsRequestSigningAlgorithms": [
-      "RS256",
-      "ES256"
+      "RS256"
```

Comment:
Should this be PS256 as well?

Reply:
I think for the consent signatures RS256 it's the correct algorithm.